File: trail0/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String[] arr = {"apple", "banana", "grape", "blueberry", "orange"};
        char a = sc.next().charAt(0);
        int cnt = 0;
        for (int i = 0; i < 5; i++) {
            if (a == arr[i].charAt(2) || a == arr[i].charAt(3)) {
                System.out.println(arr[i]);
                cnt++;
            }
        }
        System.out.print(cnt);

    }
}

File: trail1/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.cpp
#include <iostream>
using namespace std;

int main() {
    char s[5][10] {"apple", "banana", "grape", "blueberry", "orange"};
    char c;
    cin >> c;
    int cnt = 0;
    for (int i = 0; i < 5; i++) {
        if (s[i][2] == c || s[i][3] == c) {
            cout << s[i] << endl;
            cnt++;
        }
    }
    cout << cnt;
    return 0;
}